src/js/views/patients/sidebar/action/action-sidebar-action_views.js

```diff
@@ -181,14 +183,22 @@ const ActionView = View.extend({
     this.showChildView('save', new SaveView({ model: this.clonedAction }));
   },
   onSave() {
+    this.isEditingDetails = false;
+
     this.getRegion('save').empty();
   },
```

Comment:
Could this function just be:

```js
onSave() {
  this.showEditForm();
}
```

`showEditForm()` calls `this.getRegion('save').empty()`.

Reply:
so `showEditForm` immediately clones the action.. I think that would occur prior to the onSave in the app which copies the details from the clone to the model.. so saving would revert the data I think?

Reply:
_but_ it probably does make sense to put these two lines in some for of "stopEditing" that is called both here and within showEditForm

Reply:
Agreed, I added a `stopEditingDetails()`